test/functional/test-element-stub.js

```diff
@@ -15,30 +15,35 @@
  */
 
 import {ElementStub, resetLoadingCheckForTests} from '../../src/element-stub';
-import {
-    createIframePromise,
-    doNotLoadExternalResourcesInTest,
-} from '../../testing/iframe';
+import {createIframePromise} from '../../testing/iframe';
 import '../../extensions/amp-ad/0.1/amp-ad';
 import '../../extensions/amp-analytics/0.1/amp-analytics';
+import {installPerformanceService,} from
```

Comment:
nah, leftover from the previous PR (which I add performance report).

remove `doNotLoadExternalResourcesInTest` because I want the integration.js to send `render-start` to resolve `layoutCallback`